src/Http/WordPress/WordPressRequestClient.php

```diff
@@ -47,6 +47,16 @@ public function post(string $uri, $body, ?RequestOptions $options = null): Respo
         return $this->handleResponse($response);
     }
 
+    public function delete(string $uri, ?RequestOptions $options = null): Response
```

Comment:
Deze method voegt niks toe t.o.v. `get()`. Waarom heb je deze aangemaakt?

Reply:
Als het gaat om de HTTP verb, zet die standaard in deze method (`$options->set('method', 'DELETE')`) ipv in `ZakenEndpoint`

Reply:
Done.